### PR TITLE
Remove feature to link order to an existing ticket

### PR DESCRIPTION
**Pull request by ahdamin**
## Description
It was agreed by sysdev and prod to remove the option to link an existing ticket to an order in the order portal. This PR implements the backend changes.

### Changed

- Remove the feature to link orders to an existing ticket

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_cg -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

**Review by ChrOertlin — APPROVED** [review]
💯

File: cg/server/endpoints/orders.py [hunk 1]
```diff
@@ -183,7 +177,6 @@ def submit_order(order_type):
 
     except (  # user misbehaviour
         OrderError,
-        OrderExistsError,
```
Comment (diitaz93) [inline cg/server/endpoints/orders.py:186]: I think these are the only usages of this exception. Consider removing it

File: cg/meta/orders/api.py [hunk 2]
```diff
@@ -43,18 +41,8 @@ def submit(self, project: OrderType, order_in: OrderIn, user_name: str, user_mai
...
         submit_handler = self.submitter_registry.get_order_submitter(project)
         submit_handler.order_validation_service.validate_order(order_in)
-        # detect manual ticket assignment
-        ticket_number: str | None = self.ticket_handler.parse_ticket_number(order_in.name)
-        if not ticket_number:
-            ticket_number = self.ticket_handler.create_ticket(
-                order=order_in, user_name=user_name, user_mail=user_mail, project=project
-            )
-        else:
-            self.ticket_handler.connect_to_ticket(
```
Comment (diitaz93) [inline cg/meta/orders/api.py:53]: this method https://github.com/Clinical-Genomics/cg/blob/fc11fd53797c25ac091454b9a700f3bb2099eb5b/cg/meta/orders/ticket_handler.py#L192 will not be used anywhere, you could remove it from the ticket handler

Comment (diitaz93) [reply cg/meta/orders/api.py:53]: same for https://github.com/Clinical-Genomics/cg/blob/fc11fd53797c25ac091454b9a700f3bb2099eb5b/cg/meta/orders/ticket_handler.py#L29

**Review by islean — APPROVED** [review]
Looks good to me!

**Comment by ahdamin** [comment]
Tested with FE [PR](https://github.com/Clinical-Genomics/clinical-genomics-ui/pull/623).
Checkbox was removed
Screenshot 2024-12-13 at 10 40 15

Order with the following details was successfully submitted:
```
2024-12-13 10:43:09 172.18.0.1 - - [13/Dec/2024:09:43:09 +0000] "OPTIONS /api/v1/submit_order/balsamic HTTP/1.1" 204 0 "https://localhost:3001/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36"
2024-12-13 10:43:09 [2024-12-13 09:43:09 +0000] [12] [DEBUG] POST /api/v1/submit_order/balsamic
2024-12-13 10:43:09 2024-12-13 09:43:09 20b0307265fd cg.server.endpoints.orders[12] INFO processing order: {"customer": "cust000", "name": "ORDER_TEST_ISSUE_ 4013", "samples": [{"name": "SampleTestingIssue4013", "sex": "male", "subject_id": "Subject4013", "application": "EXOKTTR040", "elution_buffer": "Nuclease-free water", "source": "blood", "container": "Tube", "volume": 120, "data_analysis": "balsamic", "data_delivery": "no-delivery", "family_name": "TestingIssue4013", "cohorts": [], "phenotype_groups": [], "phenotype_terms": [], "priority": "research"}]}
2024-12-13 10:43:10 2024-12-13 09:43:10 20b0307265fd cg.meta.orders.ticket_handler[12] INFO 1001608: opened new ticket in Freshdesk
2024-12-13 10:43:10 2024-12-13 09:43:10 20b0307265fd cg.apps.lims.order[12] INFO STG15713: created new LIMS project
2024-12-13 10:43:11 2024-12-13 09:43:11 20b0307265fd cg.services.orders.submitters.order_submitter[12] INFO SampleTestingIssue4013 -> STG15713A1: connect sample to LIMS
```
Screenshot 2024-12-13 at 10 43 18

**Pull request merged**